temporal-opentracing/src/main/java/io/temporal/opentracing/internal/OpenTracingWorkflowOutboundCallsInterceptor.java

```diff
@@ -51,13 +126,16 @@ public OpenTracingWorkflowOutboundCallsInterceptor(
   @Override
   public <R> ActivityOutput<R> executeActivity(ActivityInput<R> input) {
     if (!WorkflowUnsafe.isReplaying()) {
+      Span capturedSpan = tracer.scopeManager().activeSpan();
```

Comment:
There are a lot of futures in the interceptor that are not wrapped, I will need to go and wrap them, that is one big drawback with this approach, but given the other constraints I can't think of a better approach.